android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -1363,15 +1364,21 @@ public static void registerPrimaryVideoView(PatchedVideoView v, String trackSid)
 
     public static void registerThumbnailVideoView(PatchedVideoView v) {
         thumbnailVideoView = v;
-        videoProcessor.setVideoView(v);
```

Comment:
What exactly was wrong with this approach before? Or maybe could you explain at a high level about the fix since I don't have a lot of context here.

Reply:
this approach worked until i rebased so i wish i had a better answer :(

Reply:
Puzzling... but I guess if this gets us working for now then that's an improvement.